erpnext/stock/report/itemwise_recommended_reorder_level/itemwise_recommended_reorder_level.py

```diff
@@ -55,14 +55,28 @@ def get_item_info(filters):
 
 
 def get_consumed_items(condition):
+	stock_entry_types_to_exclude = ', '.join([f"'{se}'" for se in [
+		"Material Transfer for Manufacture",
+		"Material Transfer",
+		"Send to Subcontractor"
+	]])
+	condition += f"""
+		and (
+			stock_entry_type is NULL
+			or stock_entry_type not in ({stock_entry_types_to_exclude})
+		)
+	"""
+	condition = condition.replace("posting_date",
+					"`tabStock Ledger Entry`.posting_date")
 	consumed_items = frappe.db.sql("""
 		select item_code, abs(sum(actual_qty)) as consumed_qty
-		from `tabStock Ledger Entry`
-		where actual_qty < 0
+		from `tabStock Ledger Entry` left join `tabStock Entry`
+			on `tabStock Ledger Entry`.voucher_no = `tabStock Entry`.name
+		where
+			actual_qty < 0
 			and voucher_type not in ('Delivery Note', 'Sales Invoice')
 			%s
-		group by item_code
-	""" % condition, as_dict=1)
+		group by item_code""" % condition, as_dict=1)
 
 	consumed_items_map = {}
```

Comment:
Not related to your changes but this can probably be converted to single-line dict comprehension, which is easier to read and faster.